src/isIsogram.test.js

```diff
@@ -6,4 +6,20 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true if isIsogram('')`, () => {
+    expect(isIsogram === true);
```

Comment:
```suggestion
    expect(isIsogram === true);
```
have you run your tests? isIsogram is a function and cant work without ()

Reply:
I checked everything and everything works. What needs to be fixed?